nujo/autodiff/_functions/_trigonometric.py

```diff
@@ -0,0 +1,71 @@
+from numbers import Number
+from typing import List, Union
+
+from numpy import cos, ndarray, sin, tan
+
+from nujo.autodiff.function import Function
+from nujo.autodiff.tensor import Tensor
+
+__all__ = ['_Sin', '_Cos', '_Tan']
+
+# ====================================================================================================
+
+
+class _Sin(Function):
+    ''' Differable sine function
+
+    sin(X) = (e^iX - e^-iX) / 2i
+
+    '''
+    def __init__(self, input: Union[Tensor, ndarray, List[Number], Number]):
+        super(_Sin, self).__init__(input)
+
+    def forward(self) -> ndarray:
+        return sin(self.children[0].value)
+
+    def backward(self) -> Function.T:
```

Comment:
the `backward` method signature is not like that, check develop

Reply:
Do I need to add them when they aren't used?

Reply:
yep, since the API needs to be the same for all the functions